science/osmocore/Portfile

```diff
@@ -10,10 +10,9 @@ description         ${name} provides a set of C-language libraries that form the
 long_description    ${description}
 
 categories          science comms
-license             GPL-3
+license             GPL-2+
```

Comment:
Can you remove this change & the comment change please? They have nothing to do with the main commit message. I'll make those changes once this is merged. Once that's done: LGTM & thx!

Reply:
Done in a4c0534fe08